### ISSUE TITLE
Rename refactoring of ValidationUtils and its methods to follow "Bindings" precedent in JavaFX

### ISSUE THREAD

**Issue by pfurbacher** [jidesoft/jidefx-oss#5]
ValidationUtils, and its methods, install() and uninstall(), don't seem to be suitable names to me. I would prefer that the naming follow the JavaFX naming in this regard -- _Bindings_, _bind_, and _unbind_. The renaming refactoring (this is still a beta release?) would result in the following names: 

ValidationBindings [replaces "ValidationUtils"]
bind(...) [replaces "install()"]
unbind(...) [replaces "uninstall()"]

I'm not familiar with Jide Swing libraries, but if the argument is that "we used the names 'Utils' and 'install/uninstall' in those libraries", it doesn't fly with me. Naming is always a touchy subject, but consider this: JideFX is a new library; it's for JavaFX, not Swing, and it's time for more appropriate naming, one which follows the Binding naming precedent in JavaFX. Because, essentially, that's what you are doing here -- binding and unbinding, not installing and uninstalling.

**Comment by aalmiray** [comment]
I agree with @jidesoft. The behavior provided by `install`/ `uninstall` is not related to JavaFX bindings per-se. Using any form of `bind` / `unbind` would be rather confusing IMHO. Alternative names could be `apply`/ `unapply`;  `register` / `unregister` / `deregister`.
